data/material-model/entropy-table/opxtable/header_adiabat.txt

```diff
@@ -0,0 +1,7 @@
+# This is an exemplary isentrope for an opx mantle composition, exported
```

Comment:
Why do you need this header file?

Reply:
Because whenever I get an original data file I need to attach this header, and want to keep it around. Other material models have something like this too (latent-heat-enthalpy-test).